Adjust/adjust/src/main/java/com/adjust/sdk/ResponseData.java

```diff
@@ -5,7 +5,7 @@
 /**
  * Adjust SDK
  * Created by Pedro Silva (@nonelse) on 3rd December 2015.
- * Copyright © 2015-2018 Adjust GmbH. All rights reserved.
+ * Copyright \u00a9 2015-2018 Adjust GmbH. All rights reserved.
```

Comment:
I think this broke at one point...

Reply:
It was a necessary change to start using gradle instead of maven to generate the javadocs. Is it impacting something else?